package.json

```diff
@@ -112,8 +44,12 @@
     "gcm"
   ],
   "main": "index",
+  "files": [
+    "index.js",
+    "lib/"
+  ],
   "scripts": {
-    "test": "sh node_modules/.bin/mocha -R spec test/**/*Spec.js"
+    "test": "mocha test/**/*Spec.js"
```

Comment:
The tests refused to run for me using OSX and `sh`. `spec` is also default reporter.